src/lib.rs

```diff
@@ -94,7 +94,7 @@ pub struct AxisData {
 
 #[derive(Debug)]
 pub struct ADCData {
-    pub channel: [i16; 4],
+    pub channel: [f32; 4],
```

Comment:
from the code, it's not clear why it moved to float, can you explain it better ?

Reply:
The PR intends to improve adc methods, we change the raw value to a converted value, thats uses uV per LSB.
Think it's better to have the value converted, like we are doing for all the other sensors.